src/pages/observations1/index.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/jsx-key */
```

Comment:
not sure why kept getting lint error for key prop despite adding the key so added disabled the lint rule for this file!